src/resources/js/admin/notice-install-event-tickets.js

```diff
@@ -53,6 +53,7 @@ tribe.events.admin.noticeInstall = {};
 	obj.ready = function() {
 		wp.hooks.addAction(
 			'stellarwp_installer_tec_error',
+			'event-tickets',
```

Comment:
I think we want to have this be the namespace of the addAction() call and not the namespace of the activity/description. Maybe follow this format:

```suggestion
			'tec_events',
```

Reply:
Or:

```suggestion
			'tec/events',
```